### ISSUE TITLE
Required checks are stuck at pending with `auto_fix`

### ISSUE THREAD

**Issue by dgokcin** [wearerequired/lint-action#467]
Hello and thank you for this great action

Since GITHUB_TOKEN is designed for not triggering other workflows, if there is a fix done by the linter, the required checks are stuck at a pending state. I tried using the following snippet which creates a temporary token from a GitHub app with required scopes but it is not triggering the workflow although I successfully use the same snippet to trigger other workflows.

Is there a workaround for this?

Thanks!

```
  lint:
    name: Lint
    if: ${{ !github.event.pull_request.draft }}
    runs-on: ubuntu-latest
    strategy:
      matrix:
        node-version: [14.x]
    steps:
    - name: Get Token
      id: get_workflow_token
      uses: peter-murray/workflow-application-token-action@v1
      with:
        application_id: ${{ secrets.APPLICATION_ID }}
        application_private_key: ${{ secrets.APPLICATION_PRIVATE_KEY }}

    - name: Check Out Repo
      uses: actions/checkout@v2

    - name: Use Node.js ${{ matrix.node-version }}
      uses: actions/setup-node@v3
      with:
        node-version: ${{ matrix.node-version }}

    - name: Install dependencies
      run: yarn install

    - name: Run linters
      uses: wearerequired/lint-action@v2
      with:
        github_token: ${{ steps.get_workflow_token.outputs.token }}
        eslint: true
        prettier: true
        auto_fix: true
```

**Comment by ocean90** [comment]
What happens if you use the same token for `actions/checkout`?
